securedrop/source_templates/lookup.html

```diff
@@ -28,7 +28,7 @@ <h1 class="headline">{{ gettext('Submit Materials') }}</h1>
   <input name="csrf_token" type="hidden" value="{{ csrf_token() }}">
   <div class="snippet">
     <div class="attachment grid-item center">
-      <img class="center" src="{{ url_for('static', filename='i/server_upload.png') }}" width="73px" height="62px">
+      <img class="center" src="{{ url_for('static', filename='i/font-awesome/cloud-upload-blue.png') }}" width="80px" height="56px">
```

Comment:
So this `i/font-awesome/cloud-upload-blue.png` image we have in the repo is very tiny, only 68 pixels x 47 pixels, so we'd want to add a higher resolution version to the repository so that it's not stretched, right now if you zoom it's a wee bit pixellated:

![screen shot 2018-06-18 at 9 51 51 am](https://user-images.githubusercontent.com/7832803/41551130-cda4bed8-72df-11e8-8d2a-b7a6ca7b7b0f.png)

Reply:
Thanks for all the feedback, @redshiftzero! I'll switch this image out for a higher res one, likely from the site you linked me for getting PNGs as I only see a SVG version on font awesome's website (at least without a pro account).